distribution/docs/src/main/resources/content/_integrating/_endpoints/query-endpoints.adoc

```diff
@@ -0,0 +1,241 @@
+
+* Create query metacards and store them in the ${branding} catalog.
+* Retrieve all query metacards stored in the ${branding} catalog and sort them based on attribute and sort order.
+* Retrieve a specific query metacard stored in the ${branding} catalog.
+* Update query metacards that are stored in the ${branding} catalog.
+* Delete query metacards that are stored in the ${branding} catalog.
+
+.Queries Endpoint URL
+[source,http]
+----
+https://<HOSTNAME>:<PORT>/search/catalog/internal/queries
+----
+
+=== Queries Endpoint Create Examples
+
+To create a query metacard through the queries endpoint, send a `POST` request to the queries endpoint.
+
+.Queries Endpoint Create Request Body
+[source,json]
+----
+{
+	"cql":"(\"anyText\" ILIKE 'foo bar')",
+	"filterTree":"{\"type\":\"AND\",\"filters\":[{\"type\":\"ILIKE\",\"property\":\"anyText\",\"value\":\"foo bar\"}]}",
+	"federation":"enterprise",
+	"sorts":[
+		{
+			"attribute":"modified",
+			"direction":"descending"
+		}
+	],
+	"type":"advanced",
+	"title":"Search Title"
+}
+----
+
+A successful create request will return a status of `201 CREATED`.
+
+.Queries Endpoint Create Success Response Body
+[source,json]
+----
+{
+    "id": "12bfc601cda449d58733eacaf613b93d",
+    "title": "Search Title",
+    "created": "Apr 18, 2019 10:20:55 AM",
+    "modified": "Apr 18, 2019 10:20:55 AM",
+    "owner": "admin@localhost.local",
+    "cql": "(\"anyText\" ILIKE 'foo bar')",
+    "filterTree": "{\"type\":\"AND\",\"filters\":[{\"type\":\"ILIKE\",\"property\":\"anyText\",\"value\":\"foo bar\"}]}",
+    "enterprise": null,
+    "sources": [],
+    "sorts": [
+        {
+            "attribute": "modified",
+            "direction": "descending"
+        }
+    ],
+    "polling": null,
+    "federation": "enterprise",
+    "type": "advanced",
+    "detailLevel": null,
+    "schedules": [],
+    "facets": []
+}
+----
+
+An unsuccessful create request will return a status of `500 SERVER ERROR`.
+
+.Queries Endpoint Create Failure Response Body
+[source,json]
+----
+{
+    "message": "Something went wrong."
+}
+----
+
+=== Queries Endpoint Retrieve All Examples
+
+To retrieve a query metacard through the queries endpoint, send a `GET` request to the queries endpoint.
+
+.Path Parameters
+[cols="2m,2m,2m,2m,2m" options="header"]
+|===
+
+|Query Param
+|Description
+|Default Value
+|Valid Values
+|Type
+
+|start
+|The starting index of the query to receive.
+|1
+|Integer
+|[1, 2^31)
+
+|count
+|The number of queries to return.
+|100
+|Integer
+|All integers
+
+|attr
+|The attribute to sort the queries by.
+|modified
+|All strings
+|String
+
+|sort_by
+|The sort order to return the queries in.
+|desc
+|asc, desc
+|String
+
+|===
+
+A successful retrieval request will return a status of `200 OK`.
+
+=== Queries Endpoint Retrieve Examples
+
+.Retrieve Specific Query Endpoint URL
+[source,http]
+----
+https://<HOSTNAME>:<PORT>/search/catalog/internal/queries/:id
```

Comment:
We should offset the placeholder `id` with angle brackets, like `<HOSTNAME>` and `<PORT>` 
and, I suppose, do all caps, although I don't think I've been very consistent on the other endpoints.